psycop/projects/sequence_models/fine_tune.py

```diff
@@ -0,0 +1,106 @@
+import datetime as dt
+from pathlib import Path
+from typing import Any
+
+from pydantic import BaseModel, FilePath
+from torch.utils.data import DataLoader
+
+from psycop.common.feature_generation.loaders.raw.load_ids import SplitName
+from psycop.common.feature_generation.sequences.cohort_definer_to_prediction_times import (
+    CohortToPredictionTimes,
+)
+from psycop.common.feature_generation.sequences.patient_loaders import (
+    DiagnosisLoader,
+    PatientLoader,
+)
+from psycop.common.sequence_models.aggregators import AggregationModule, AveragePooler
+from psycop.common.sequence_models.dataset import PatientSlicesWithLabels
+from psycop.common.sequence_models.tasks import (
+    BEHRTForMaskedLM,
+    EncoderForClassification,
+)
+from psycop.projects.t2d.feature_generation.cohort_definition.t2d_cohort_definer import (
+    T2DCohortDefiner,
+)
+
+
+class DataCfg(BaseModel):
+    lookbehind: dt.timedelta
+    lookahead: dt.timedelta
+    num_classes: int
```

Comment:
I don't think these will get logged to wandb

Reply:
it is for another PR though

Reply:
Reference in https://github.com/Aarhus-Psychiatry-Research/psycop-common/issues/286